riva/clients/asr/riva_asr_client_helper.cc

```diff
@@ -108,3 +108,85 @@ CloseAudioDevice(snd_pcm_t** handle)
   }
   return true;
 }
+
+void
+AppendResult(
```

Comment:
This will just combine the separated utterances into one transcript.  Doesn't that defeat the purpose?